### PR TITLE
feat: add one-click deploy script for HyperSwitch on AWS (EC2, RDS, Redis)

### PR DESCRIPTION
**Pull request by BallaNitesh**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description
This script streamlines the deployment of the HyperSwitch stack on AWS, encapsulating the entire setup within a single click. Utilizing minimal configurations, the script orchestrates the creation of a single EC2 instance, an RDS database, and a Redis cluster on AWS.

### Additional Changes

- [ ] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

## Checklist

- [ ] I formatted the code `cargo +nightly fmt --all`
- [ ] I addressed lints thrown by `cargo clippy`
- [x] I reviewed the submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

File: aws/schema.sql
Comment (NishantJoshi00) [inline aws/schema.sql:1]: can't we have a computational logic instead of using this file.

cc: @SanchithHegde

**Review by NishantJoshi00 — CHANGES_REQUESTED** [review]
other than that, looks good!

File: aws/schema.sql
Comment (BallaNitesh) [reply aws/schema.sql:1]: Yes, For every open source release, we'll publish a schema.sql specific to that version. For the beta, let's use the hardcoded version and rename the file to schema-beta.sql.

Comment (inventvenkat) [reply aws/schema.sql:1]: We can remove this from main, once we setup the flow.

**Review by inventvenkat — DISMISSED** [review]

**Review by inventvenkat — APPROVED** [review]

**Review by NishantJoshi00 — APPROVED** [review]

**Review by inventvenkat — APPROVED** [review]

**Pull request merged**